docs/reference/commandline/network_create.md

```diff
@@ -41,12 +41,14 @@ Options:
 ## Description
 
 Creates a new network. The `DRIVER` accepts `bridge` or `overlay` which are the
-built-in network drivers. If you have installed a third party or your own custom
+built-in network drivers on Linux. Windows has built-in network drivers `nat`, 
+`transparent`, and `overlay`. If you have installed a third party or your own custom
 network driver you can specify that `DRIVER` here also. If you don't specify the
-`--driver` option, the command automatically creates a `bridge` network for you.
-When you install Docker Engine it creates a `bridge` network automatically. This
+`--driver` option, the command automatically creates a `bridge` network for you on Linux, or `nat` on Windows.
```

Comment:
Can you wrap this to 80-chars?

    `--driver` option, the command automatically creates a `bridge` network for you
    on Linux, or `nat` on Windows.